content/docs/1.36/deployment.md

```diff
@@ -667,6 +667,17 @@ docker run \
   jaegertracing/all-in-one:{{< currentVersion >}}
 ```
 
+#### Available remote gRPC service:
+
+* [Promscale](https://github.com/timescale/promscale#promscale-for-jaeger-and-opentelemetry) - observability backend for metrics and traces powered by SQL and built on PostgreSQL and TimescaleDB.
+
+```sh
+docker run \
+  -e SPAN_STORAGE_TYPE=grpc-plugin \
+  -e GRPC_STORAGE_SERVER=<...> \
+  jaegertracing/all-in-one:{{< currentVersion >}} 
+```
```

Comment:
Do you think we need this here?

Reply:
I prefer to have this code samples in the promscale docs rather than here (we should also remove the clickhouse example).